web/modules/custom/portland/portland.module

```diff
@@ -1477,3 +1457,56 @@ function portland_form_user_form_alter(&$form, FormStateInterface $form_state) {
     unset($form['account']['saml_notice']);
   }
 }
+
+/**
+ * Implements hook_media_presave().
+ */
+function portland_media_presave($media)
+{
+  // Get file size and mime type from eFiles link
+  if($media->hasField('field_efiles_link')) {
+    if (!empty($media->field_efiles_link) && !empty($media->field_efiles_link[0]->uri)) {
+      $efiles_link = $media->field_efiles_link[0]->uri;
+      $user_input_file_url = strtolower($efiles_link);
```

Comment:
`$user_input_file_url` Appears to be unused in this function, so why not just assign this directly to `$external_file_url`?

Reply:
Removed the variable `$user_input_file_url`